plugins/plugin-testing-java/pom.xml

```diff
@@ -26,6 +26,15 @@
         <module>plugin-testing-junit</module>
         <module>plugin-testing-testng</module>
     </modules>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.github.jnr</groupId>
+                <artifactId>jnr-posix</artifactId>
+                <version>3.0.38</version>
```

Comment:
introduction of specific version should be done at https://github.com/eclipse/che-dependencies/blob/master/pom.xml  (there is also an openshift branch)